src/oncall_admin/ldap_user.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) LinkedIn Corporation. All rights reserved. Licensed under the BSD-2 Clause license.
+# See LICENSE in the project root for license information.
+
+import ldap
+from oncall import db
+import os
+import logging
+
+logger = logging.getLogger(__name__)
+
```

Comment:
Initialize globals to None. This way, `get_ldap_user()` avoids a potential NameError on line 40, as `connection` may be undefined.

Reply:
OK, I've initialized all globals to None.
The `get_ldap_user` function returns None if `connection` is not set.